FF1Lib/Interfaces.cs

```diff
@@ -149,6 +149,8 @@ public interface IFloorShuffleFlags
 		bool? DeepTownsPossible { get; }
 		bool? AllowDeepTowns { get; }
 		bool? AllowUnsafeStartArea { get; }
+		bool? NoFloater { get; }

```

Comment:
I'd suggest using the internal flag IsFloaterRemoved instead of NoFloater; while it doesn't change anything right now, it might simplify things down the road if other flags start removing the floater

Reply:
That doesn't work.  `IsFloaterRemoved` is a field on `IVictoryConditionFlags` which is used by item placement, but the code that checks `NoFloater` is `ShuffleEntrancesAndFloors` which uses `IFloorShuffleFlags`, not `IVictoryConditionFlags`.

Reply:
Unless you're saying I should just literally rename the field to ` IsFloaterRemoved` but that doesn't work either because it just passes in the `Flags` object which implements `IFloorShuffleFlags` so the field name has to match.

Reply:
oh, I just meant to replace `NoFloater` with `IsFloaterRemoved` in `IFloorShuffleFlags`, then you get access to it in `ShuffleEntranceAndFloors`, the compiler will only get confused if several interfaces are merged in a single one, but then you can precise the one you're using by putting he target interface in () before like:
`(IFloorShuffleFlags)flags.IsFloaterRemoved`

Reply:
Oh, I see now.  I missed that.  Will fix.